lib/webpacker/helper.rb

```diff
@@ -42,8 +45,25 @@ def javascript_pack_tag(*names, **options)
   #   # In production mode:
   #   <%= stylesheet_pack_tag 'calendar', 'data-turbolinks-track': 'reload' %> # =>
   #   <link rel="stylesheet" media="screen" href="/packs/calendar-1016838bab065ae1e122.css" data-turbolinks-track="reload" />
+  #   # In development mode with hot-reloading
+  #   <%= stylesheet_pack_tag('main') %> <% # Default is false for enabled_when_hot_loading%>
+  #   # No output
+  #
+  #   # In development mode with hot-reloading and enabled_when_hot_loading
+  #   # <%= stylesheet_pack_tag('main', enabled_when_hot_loading: true) %>
+  #   <link rel="stylesheet" media="screen" href="/public/webpack/development/calendar-1016838bab065ae1e122.css" />
+  #
   def stylesheet_pack_tag(*names, **options)
-    sources = names.map { |name|  Webpacker::Manifest.lookup("#{name}#{compute_asset_extname(name, type: :stylesheet)}") }
-    stylesheet_link_tag(*sources, **options)
+    return "" if Webpacker::DevServer.hot?
+    stylesheet_link_tag(*asset_source(names, :stylesheet), **options)
```

Comment:
That's fine. However, the default for Rubocop is the [Guard Clause](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/GuardClause).

In eslint, we have [Disallow return before else (no-else-return)](http://eslint.org/docs/rules/no-else-return) and the airbnb eslint pattern is to use error for this. Thus, for those of us that do lots of JS with the airbnb linter, we're used to the pattern of the "guard clause".

That being said, since there's no `return` needed in Ruby, I think your suggestion is fine, and it's 💯 what I used to always do and have the linter complain to me.